![Illustration of silhouetted heads](mentalhealth.jpg)

Does going to university in a different country affect your mental health? A Japanese international university surveyed its students in 2018 and published a study the following year that was approved by several ethical and regulatory boards.

The study found that international students have a higher risk of mental health difficulties than the general population, and that social connectedness (belonging to a social group) and acculturative stress (stress associated with joining a new culture) are predictive of depression.


Explore the `students` data using PostgreSQL to find out if you would come to a similar conclusion for international students and see if the length of stay is a contributing factor.

Here is a data description of the columns you may find helpful.

| Field Name    | Description                                      |
| ------------- | ------------------------------------------------ |
| `inter_dom`     | Types of students (international or domestic)   |
| `japanese_cate` | Japanese language proficiency                    |
| `english_cate`  | English language proficiency                     |
| `academic`      | Current academic level (undergraduate or graduate) |
| `age`           | Current age of student                           |
| `stay`          | Current length of stay in years                  |
| `todep`         | Total score of depression (PHQ-9 test)           |
| `tosc`          | Total score of social connectedness (SCS test)   |
| `toas`          | Total score of acculturative stress (ASISS test) |

Let's explore and analyze the _students_ data to see how the length of stay(_stay_) impacts the average mental health diagnostic scores of the International students present in the study.

Let's check the table to clearly understand it rows and fields

In [24]:
-- Run this code to view the data in students
SELECT * 
FROM students
LIMIT 5;

inter_dom region  gender academic  ...  religion_bi  alone_bi  others_bi internet_bi
0     Inter    SEA    Male     Grad  ...           No        No         No          No
1     Inter    SEA    Male     Grad  ...           No        No         No          No
2     Inter    SEA    Male     Grad  ...           No        No         No          No
3     Inter     EA  Female     Grad  ...           No        No         No          No
4     Inter     EA  Female     Grad  ...          Yes        No         No          No

[5 rows x 50 columns]

Next we will slowly creating our fields or columns piece by piece, let's start by performing the calculations. Let's find the summary statistics for each diagnostic test using aggregate functions.

In [25]:
SELECT 
	COUNT(*) AS count_int, 
	ROUND(AVG(todep), 2) AS average_phq, 
	ROUND(AVG(tosc), 2) AS average_scs,
	ROUND(AVG(toas), 2) AS average_as
FROM students
WHERE inter_dom = 'Inter';

count_int  average_phq  average_scs  average_as
0        201         8.04        37.42       75.56

Since we've previously performed counts and average calculations on the data; now we need to apply the appropriate filter and group so that the calculations are done on the international student group only.

In [26]:
SELECT 
	stay,
	COUNT(*) AS count_int, 
	ROUND(AVG(todep), 2) AS average_phq, 
	ROUND(AVG(tosc), 2) AS average_scs,
	ROUND(AVG(toas), 2) AS average_as
FROM students
WHERE inter_dom = 'Inter'
GROUP BY stay;

stay  count_int  average_phq  average_scs  average_as
0     8          1        10.00        44.00       65.00
1     7          1         4.00        48.00       45.00
2    10          1        13.00        32.00       50.00
3     1         95         7.48        38.11       72.80
4     5          1         0.00        34.00       91.00
5     4         14         8.57        33.93       87.71
6     2         39         8.28        37.08       77.67
7     6          3         6.00        38.00       58.67
8     3         46         9.09        37.13       78.00

Finally, let’s ORDER BY stay DESC to sort results by length of stay in descending order.

In [27]:
SELECT 
	stay,
	COUNT(*) AS count_int, 
	ROUND(AVG(todep), 2) AS average_phq, 
	ROUND(AVG(tosc), 2) AS average_scs,
	ROUND(AVG(toas), 2) AS average_as
FROM students
WHERE inter_dom = 'Inter'
GROUP BY stay
ORDER BY stay DESC;

stay  count_int  average_phq  average_scs  average_as
0    10          1        13.00        32.00       50.00
1     8          1        10.00        44.00       65.00
2     7          1         4.00        48.00       45.00
3     6          3         6.00        38.00       58.67
4     5          1         0.00        34.00       91.00
5     4         14         8.57        33.93       87.71
6     3         46         9.09        37.13       78.00
7     2         39         8.28        37.08       77.67
8     1         95         7.48        38.11       72.80

## Depression risk (PHQ-9):

The average PHQ scores (average_phq) are higher in the mid-to-long stays (e.g., 8–10 stay lengths have values around 10–13).

Students with shorter stays (1–2) have slightly lower PHQ averages (around 7.48–8.28).

This suggests that depression scores may increase with time for some students.

## Social connectedness (SCS):

Scores (average_scs) fluctuate but generally hover around the mid-30s to high-30s, with some very low values in longer stays (e.g., stay = 4 has 33.93).

Lower SCS values reflect weaker feelings of belonging, supporting the idea that social connectedness is linked to mental health struggles.

## Acculturative stress (ASISS):

The AS scores (average_as) show a pattern where students with shorter stays (1–2) already report high stress (72–77), and stress remains high or even spikes for some mid-stays (e.g., 87–91 at stays 4–5).

This indicates that adapting to a new culture is stressful across all groups, not just newcomers.

## ✅ Conclusion:
Based on our results, we can reasonably agree with the study: international students experience higher risks of mental health difficulties compared to general populations, and both social connectedness and acculturative stress significantly contribute to depression. Length of stay adds another dimension students often start with high stress, and their depression scores can worsen if adjustment or support is lacking.